# --- T2-COPYRIGHT-NOTE-BEGIN ---
# This copyright note is auto-generated by ./scripts/Create-CopyPatch.
# 
# T2 SDE: package/.../acx100/ld_var.patch
# Copyright (C) 2004 - 2005 The T2 SDE Project
# 
# More information can be found in the files COPYING and README.
# 
# This patch file is dual-licensed. It is available under the license the
# patched project is licensed under, as long as it is an OpenSource license
# as defined at http://www.opensource.org/ (e.g. BSD, X11) or under the terms
# of the GNU General Public License as published by the Free Software
# Foundation; either version 2 of the License, or (at your option) any later
# version.
# --- T2-COPYRIGHT-NOTE-END ---
--- ./src/Makefile.orig	2005-11-21 18:30:34.000000000 +0100
+++ ./src/Makefile	2005-11-21 18:35:07.000000000 +0100
@@ -132,7 +132,7 @@
 	$(LD) $(LDFLAGS) -o $@ $^
 
 acx_usb.o: $(usb_objs)
-	ld $(LDFLAGS) -o $@ $^
+	$(LD) $(LDFLAGS) -o $@ $^
 
 acx_usb_80211frm.o: acx80211frm.c
 	$(CC) $(USBCFLAGS) -c -o $@ $<
